relayer/relayer/src/connections/azero.rs

```diff
@@ -1,3 +1,5 @@
+use std::sync::{Arc, Mutex as StdMutex};
```

Comment:
I would called SyncMutex (since Tokio is Async/await)

Reply:
done

Reply:
changed to atomicu64